test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -72,6 +72,15 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 			if err != nil {
 				log.Printf("Error while getting Ubuntu image version: %s\n", out)
 			}
+
+			kernelVerCmd := fmt.Sprintf("cat /proc/version")
+			cmd := exec.Command("ssh", "-i", sshKeyPath, "-o", "ConnectTimeout=10", "-o", "StrictHostKeyChecking=no", "-o", "UserKnownHostsFile=/dev/null", master, kernelVerCmd)
+			util.PrintCommand(cmd)
+			out, err := cmd.CombinedOutput()
+			log.Printf("%s\n", out)
+			if err != nil {
+				log.Printf("Error while getting LinuxKernel version: %s\n", out)
```

Comment:
We already logged the `CombinedOutput` when we checked the error value. Maybe this should print the `err` instead?

Reply:
Good catch! Corrected, like, in a lot of places!